runtime/compiler/build/files/target/amd64.mk

```diff
@@ -34,3 +34,12 @@ JIT_PRODUCT_SOURCE_FILES+=\
     compiler/x/amd64/codegen/AMD64JNILinkage.cpp \
     compiler/x/amd64/codegen/AMD64PrivateLinkage.cpp \
     compiler/x/amd64/codegen/J9CodeGenerator.cpp
+
+ifneq ($(J9VM_OPT_MICROJIT),)
+JIT_PRODUCT_SOURCE_FILES+=\
+    compiler/microjit/x/amd64/AMD64Codegen.cpp \
+    compiler/microjit/x/amd64/AMD64CodegenGC.cpp \
+    compiler/microjit/x/amd64/AMD64Linkage.cpp \
+    compiler/microjit/x/amd64/templates/linkage.nasm \
+    compiler/microjit/x/amd64/templates/bytecodes.nasm
+endif
```

Comment:
The line-end is missing here.

Reply:
Thank you, I've updated all file-endings to have the required new-line at EOF.